### PR TITLE
samples: Bluetooth: Update Readme of large MTU sample

### PR DESCRIPTION
**Pull request by theob-pro**
Update the Readme file of the large mtu sample because the orginal PR (https://github.com/zephyrproject-rtos/zephyr/pull/53593) has been merged before the last comment (https://github.com/zephyrproject-rtos/zephyr/pull/53593#discussion_r1070969497) could be addressed.

### CODE REVIEW

File: samples/bluetooth/mtu_update/README.rst [hunk 1]
```diff
@@ -66,7 +66,8 @@ This sample use two applications, two devices need to be setup.
 The first one should be flashed with the central and the second one with the
 peripheral.
 
-The two devices need to be close enough to be able to connect.
+The two devices will connect only if they are close to each other, thanks to
```
Comment (jori-nordic) [inline samples/bluetooth/mtu_update/README.rst:69]: s/thanks/because

**Comment by Thalley** [comment]
Why are we adding this text only specifically for this sample? A lot of our connection samples have RSSI filtering

**Comment by alwa-nordic** [comment]
> Why are we adding this text only specifically for this sample? A lot of our connection samples have RSSI filter

We are just clarifying text that is already there.

**Comment by Thalley** [comment]
> > Why are we adding this text only specifically for this sample? A lot of our connection samples have RSSI filter
> 
> We are just clarifying text that is already there.

Ah, sorry, had missed that this sample already had text w.r.t. this.

**Review by Thalley — APPROVED** [review]

**Review by jhedberg — APPROVED** [review]

**Pull request merged**